pages/02.Tutorials/10.cinder-multiattach/docs.en.md

```diff
@@ -0,0 +1,161 @@
+| replication_status  | None                                 |
+| size                | 50                                   |
+| snapshot_id         | None                                 |
+| source_volid        | None                                 |
+| status              | creating                             |
+| type                | quobyte-multiattach                  |
+| updated_at          | None                                 |
+| user_id             | 8e9e69c9d85e4b4a82382b9e9e0f59c1     |
++---------------------+--------------------------------------+
+```
+
+### Attach the new volume to the instances
+
+For this tutorial the following 3 instances are used:
+
+```shell
+openstack server list
++--------------------------------------+----------------+--------+----------------------------------------+----------------------------------+
+| ID                                   | Name           | Status | Networks                               | Image Name                       |
++--------------------------------------+----------------+--------+----------------------------------------+----------------------------------+
+| 2befeba0-70f1-4049-97d8-98a71a3b6d72 | App Instance 1 | ACTIVE | net_demo=192.168.1.14                | Ubuntu Xenial 16.04 (2020-03-30) |
+| 6e607706-13e2-4433-8665-3617835865c0 | App Instance 2 | ACTIVE | net_demo=192.168.1.2                 | Ubuntu Xenial 16.04 (2020-03-30) |
+| a6d64bf4-d2da-4d17-ada0-9b5c10942bc9 | App Instance 3 | ACTIVE | net_demo=192.168.1.6                 | Ubuntu Xenial 16.04 (2020-03-30) |
++--------------------------------------+----------------+--------+----------------------------------------+----------------------------------+
+
+openstack --os-compute-api-version 2.60 server add volume "App Instance 1" testvolume
+openstack --os-compute-api-version 2.60 server add volume "App Instance 2" testvolume
+openstack --os-compute-api-version 2.60 server add volume "App Instance 3" testvolume
+
+```
+
+If you get an error message like
+>versions supported by client: 2.1 - 2.41
+
+you need to update your openstack client. Working with multi attach volumes is supported from nova API version 2.60.
+
+
+### Setup OCFS2
+
+Multi-attach volumes require a [cluster filesystem](https://en.wikipedia.org/wiki/Clustered_file_system#SHARED-DISK) like OCFS2 or GFS2 to coordinate concurrent filesystem access.
+
+**Warning: We do not recommend to use a file system like Ext4 or XFS for multi-attach volumes. Concurrent write access will destroy the filesystem and the data on that volume.**
+
+For this tutorial we use OCFS2.
+To setup OCFS2 we need to install the ocfs2-tools and kernel module for the running kernel version on every VM:
+
+```shell
+apt install ocfs2-tools linux-modules-extra-4.15.0-91-generic
+```
+
+OCFS2 is configurend in /etc/default/o2cb and /etc/ocfs/cluster.conf.
+Set `O2CB_ENABLED=true` in /etc/default/o2cb. All other settings can left unchanged.
```

Comment:
...can be left unchanged. ("be" was missing)

Reply:
fixed